src/models/gbdt.py

```diff
@@ -0,0 +1,202 @@
+import numpy as np
+from pathlib import Path
+import pickle
+
+from typing import Any, Dict, List, Tuple, Optional
+
+from .base import ModelBase
+from .data import DataLoader, train_val_mask, TrainData
+
+xgb = None
+
+
+class GBDT(ModelBase):
+
+    """This only really works if you have a lot of memory
+    """
```

Comment:
```
    (all data has to be loaded into memory for the XGBRegressor)

```

Reply:
just so it's clear why a user might need alot of memory 

(user being future us ... ;) )

Reply:
Added !